horizon_bsn/api/neutron.py

```diff
@@ -24,6 +27,37 @@
 neutronclient = neutron.neutronclient
 
 
+@memoized
+def list_extensions(request):
```

Comment:
I found https://github.com/openstack/horizon/blob/master/openstack_dashboard/api/neutron.py#L1808

The codes looks identical, could we use it directly, or is there any reason we need to duplicate the codes here?

Reply:
yep, its the same check. if upstream moves api.neutron to a new location, it causes failure at our end without notice.
for minor util methods, its easier to clone and not worry about future updates and breakages

Reply:
correction - i did not realize i'm already importing api.neutron for creating the neutronclient.
i'm not saving anything by cloning the method from the same class 🤦🏻‍♂️
now uses upstream method directly 🙂